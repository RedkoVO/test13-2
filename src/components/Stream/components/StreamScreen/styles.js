export default () => ({
  root: {
    width: '990px',

    '& img': {
      width: '100%'
    }
  }
})
