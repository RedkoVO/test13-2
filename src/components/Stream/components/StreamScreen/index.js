import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import StreamImage from '../../../../assets/images/stream-image.png'

import styles from './styles'

const StreamScreen = ({ classes }) => (
  <div className={classes.root}>
    <img src={StreamImage} alt="" />
  </div>
)

StreamScreen.propTypes = {
  classes: PropTypes.object
}

export default withStyles(styles)(StreamScreen)
