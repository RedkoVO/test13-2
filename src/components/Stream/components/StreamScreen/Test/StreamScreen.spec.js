import * as React from 'react'
import renderer from 'react-test-renderer'

import StreamScreen from '../'

it('renders correctly', () => {
  const tree = renderer.create(<StreamScreen />).toJSON()
  expect(tree).toMatchSnapshot()
})
