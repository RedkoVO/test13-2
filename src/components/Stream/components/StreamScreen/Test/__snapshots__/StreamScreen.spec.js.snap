// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="StreamScreen-root-1"
>
  <img
    alt=""
    src="stream-image.png"
  />
</div>
`;
